plugins/frontity-headtags/readme.txt

```diff
@@ -16,6 +16,12 @@ This plugin adds all the tags in the head section of a website to WordPress REST
 
 It is perfect if you are using WordPress for a headless set-up and would like to add the **meta tags** generated by your **WordPress SEO plugin** (like Yoast SEO or All-in-One SEO Pack) to the WordPress REST API output.
 
+#### Requisites 
```

Comment:
@mburridge is requisites the correct word here? I'm more used to "Requirements", but I'd love to know your opinion.

Reply:
I would use either 'Requirements' or 'Prerequisites'.

Reply:
Changed.